diff --git a/node_modules/@nebular/theme/bundles/theme.umd.js b/node_modules/@nebular/theme/bundles/theme.umd.js
index e83ea41..fe28be9 100644
--- a/node_modules/@nebular/theme/bundles/theme.umd.js
+++ b/node_modules/@nebular/theme/bundles/theme.umd.js
@@ -1660,7 +1660,7 @@ var NbOverlayService = /** @class */ (function () {
 var NbScrollDispatcherAdapter = /** @class */ (function (_super) {
     tslib.__extends(NbScrollDispatcherAdapter, _super);
     function NbScrollDispatcherAdapter(ngZone, platform, scrollService) {
-        var _this = _super.call(this, ngZone, platform) || this;
+        var _this = _super.call(this, ngZone, platform, document) || this;
         _this.scrollService = scrollService;
         return _this;
     }
diff --git a/node_modules/@nebular/theme/esm2015/components/cdk/adapter/viewport-ruler-adapter.js b/node_modules/@nebular/theme/esm2015/components/cdk/adapter/viewport-ruler-adapter.js
index 2485b14..1220ab6 100644
--- a/node_modules/@nebular/theme/esm2015/components/cdk/adapter/viewport-ruler-adapter.js
+++ b/node_modules/@nebular/theme/esm2015/components/cdk/adapter/viewport-ruler-adapter.js
@@ -6,7 +6,7 @@ import { NbLayoutRulerService } from '../../../services/ruler.service';
 import { NbLayoutScrollService } from '../../../services/scroll.service';
 export class NbViewportRulerAdapter extends ViewportRuler {
     constructor(platform, ngZone, ruler, scroll) {
-        super(platform, ngZone);
+        super(platform, ngZone, document);
         this.ruler = ruler;
         this.scroll = scroll;
     }
diff --git a/node_modules/@nebular/theme/fesm2015/index.js b/node_modules/@nebular/theme/fesm2015/index.js
index 289ff0a..8f4c32b 100644
--- a/node_modules/@nebular/theme/fesm2015/index.js
+++ b/node_modules/@nebular/theme/fesm2015/index.js
@@ -959,7 +959,7 @@ NbOverlayContainerAdapter.decorators = [
 
 class NbViewportRulerAdapter extends ViewportRuler {
     constructor(platform, ngZone, ruler, scroll) {
-        super(platform, ngZone);
+        super(platform, ngZone, document);
         this.ruler = ruler;
         this.scroll = scroll;
     }
